core/new-gui/package.json

```diff
@@ -31,12 +31,14 @@
     "@ng-bootstrap/ng-bootstrap": "^4.2.1",
     "@ngx-formly/bootstrap": "^5.0.0",
     "@ngx-formly/core": "^5.5.15",
-    "@ngx-formly/schematics": "^5.5.15",
     "@ngx-formly/material": "^5.0.0",
+    "@ngx-formly/schematics": "^5.5.15",
+    "@types/c3": "^0.7.4",
```

Comment:
move `@types/c3` to `devDependencies`. all the `@types/...` are in devDependencies